pydough/unqualified/unqualified_transform.py

```diff
@@ -69,7 +102,7 @@ def visit_FunctionDef(self, node):
                 name=node.name,
                 args=node.args,
                 body=prefix + node.body,
-                decorator_list=node.decorator_list,
+                decorator_list=decorator_list,
```

Comment:
Bug fix: Since we need to remove `init_pydough_context`, the updated deco list needs to be sent.